Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -3492,6 +3493,7 @@ CSSPropertyAnimationWrapperMap::CSSPropertyAnimationWrapperMap()
         new DiscretePropertyWrapper<WhiteSpace>(CSSPropertyWhiteSpace, &RenderStyle::whiteSpace, &RenderStyle::setWhiteSpace),
         new DiscretePropertyWrapper<WordBreak>(CSSPropertyWordBreak, &RenderStyle::wordBreak, &RenderStyle::setWordBreak),
         new DiscretePropertyWrapper<OverflowAnchor>(CSSPropertyOverflowAnchor, &RenderStyle::overflowAnchor, &RenderStyle::setOverflowAnchor),
+        new DiscretePropertyWrapper<TextSpacingTrim>(CSSPropertyTextSpacingTrim, &RenderStyle::textSpacingTrim, &RenderStyle::setTextSpacingTrim),
```

Comment:
It doesn't look like there is testing coverage for this. Could you add the appropriate entry to [web-animations/animation-model/animation-types/property-list.js](https://github.com/web-platform-tests/wpt/blob/master/web-animations/animation-model/animation-types/property-list.js)? Should be something like this:

```javascript
  'text-spacing-trim': {
    // https://drafts.csswg.org/link-to-spec
    types: [
      { type: 'discrete' , options: [ [ 'space-all', 'trim-all' ] ] }
    ]
  },
```

You can feel free to expand on that of course.

Reply:
Fixed, thanks.